docs/macos_install_guide.rst

```diff
@@ -7,8 +7,7 @@ Development Setup on macOS
 
 .. :highlight: bash
 
-The following instructions will guide you from a clean installation of macOS to a working source
-checkout of raiden.
+The following instructions will guide you from a clean installation of macOS to a working source checkout of raiden. Make sure that you are on OSX 10.12 or higher. This is needed since raiden uses the MONOTONICK_CLOCK_RAW attribute as seen `here
```

Comment:
Is there a link missing at the end of this line?

Reply:
Yes. I will add it.